packages/sycamore-macro/src/view/parse.rs

```diff
@@ -248,11 +256,55 @@ impl Parse for Component {
 
 impl Parse for ComponentProp {
     fn parse(input: ParseStream) -> Result<Self> {
-        Ok(Self {
-            name: input.parse()?,
-            eq: input.parse()?,
-            value: input.parse()?,
-        })
+        pub struct AttributeName {
+            tag: Ident,
+            extended: Vec<(Token![-], Ident)>,
+        }
```

Comment:
We should probably reuse the `AttributeName` from above to prevent code-duplication, since we are parsing the exact same syntax.

Reply:
Thanks for reminding me, I wanted to do that before but forgot